css.js

```diff
@@ -50,10 +64,17 @@ if (typeof window !== 'undefined') {
       } else {
         webkitLoadCheck(link, _callback);
       }
+      
       link.onerror = function(event) {
         _callback(event.error || new Error('Error loading CSS file.'));
       };
-      head.appendChild(link);
+
+      if (present)        
+         link.href = url + '?' + new Date().getTime();
```

Comment:
I have tried removing the link and adding a new one, but it causes an unsightly flicker in the browser. Changing the href like this makes the transition smooth.
The onload event is being fired when hot-reloading so I don't think that is a problem. Looking at the code there may be an issue with the callbacks happening more than once which I can fix.